support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -162,7 +166,7 @@ test.describe('Supporter Plus promoCodes', () => {
 				card.getByText(testDetails.expectedPromoText).first(),
 			).toBeVisible();
 			await page
-				.getByRole('link', { name: 'Subscribe' })
+				.getByRole('link', { name: ctaCopy })
```

Comment:
Presume you don't want to just leave as `Support` in case the country canges to `US`?

Reply:
Yep pretty much, although we don't have regional variants of this test, we may do like we do already on the other